valen-core/src/main/java/be/twofold/valen/core/geometry/VertexBufferInfo.java

```diff
@@ -5,26 +5,27 @@
 public record VertexBufferInfo<T extends Buffer>(
     Semantic semantic,
     ComponentType<T> componentType,
-    int size
+    int size,
+    String customName
```

Comment:
Provide a default constructor instead of forcing every consumer to supply an optional parameter.